packages/smooth_app/lib/pages/onboarding/scan_example.dart

```diff
@@ -0,0 +1,100 @@
+import 'package:flutter/cupertino.dart';
+import 'package:flutter/material.dart';
+import 'package:flutter_gen/gen_l10n/app_localizations.dart';
+import 'package:flutter_svg/flutter_svg.dart';
+import 'package:smooth_app/pages/onboarding/next_button.dart';
+import 'package:smooth_app/pages/onboarding/onboarding_flow_navigator.dart';
+import 'package:smooth_ui_library/util/ui_helpers.dart';
+
+/// Example explanation on how to scan a product.
+class ScanExample extends StatelessWidget {
+  const ScanExample();
+
+  @override
+  Widget build(BuildContext context) {
+    final AppLocalizations appLocalizations = AppLocalizations.of(context)!;
+    final Size screenSize = MediaQuery.of(context).size;
+    return Stack(
+      children: <Widget>[
+        Container(
+          color: const Color.fromARGB(255, 225, 208, 208),
+          height: screenSize.height,
+        ),
+        Column(
+          children: <Widget>[
+            Flexible(
+              flex: 8,
+              child: Stack(
+                children: <Widget>[
+                  Positioned(
+                    top: -screenSize.height / 8,
+                    left: screenSize.width / 16,
+                    child: Transform.rotate(
+                      angle: -0.22828907, // 13.08 degrees
```

Comment:
The question is rather: why do you feel the desire to rotate by -13.08 degrees?

Reply:
just following UX mocks that UX agreed upon with Stephane and Pierre involved.

Reply:
Then, you can comment: "we rotate with this angle in order to correctly display the svg"
Which sounds weird as the svg is already almost systematically rotated with 13.08 degrees:
```svg
<rect x="53" y="150.35" width="112" height="58" transform="rotate(-13.0814 53 150.35)" fill="white"/>
```

Reply:
this isn't rotating the svg image, it's rotating the green container

Reply:
domment added